apis/hive/v1/machinepool_types.go

```diff
@@ -46,11 +46,19 @@ type MachinePoolSpec struct {
 	Platform MachinePoolPlatform `json:"platform"`
 
 	// Map of label string keys and values that will be applied to the created MachineSet's
-	// MachineSpec. This list will overwrite any modifications made to Node labels on an
-	// ongoing basis.
+	// MachineSpec. This affects the labels that will end up on the *Nodes* (in contrast with
+	// the MachineLabels field).This list will overwrite any modifications made to Node labels
```

Comment:
```suggestion
	// the MachineLabels field). This list will overwrite any modifications made to Node labels
```

:roll_eyes: 
If an update is needed...